### PR TITLE
Synchronize manual with stubs for PHP 8.4 - part 6

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/sqlite3/sqlite3result/finalize.xml [hunk 1]
```diff
@@ -25,10 +25,40 @@
...
   <para>
-   Returns &true;.
+   &return.true.always;.
   </para>
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <para>
+   An <exceptionname>Error</exceptionname> is thrown if the method is called with an uninitialized object.
```
Comment (cmb69) [inline reference/sqlite3/sqlite3result/finalize.xml:35]: "with" or "on"? I think the latter is more appropriate here, but I'm fine with the wording if we already use it elsewhere.

**Review by cmb69 — COMMENTED** [review]
Thank you!

File: reference/sqlite3/sqlite3result/finalize.xml (hunk 1 above)
Comment (kocsismate) [reply reference/sqlite3/sqlite3result/finalize.xml:35]: I'm not sure, but after reading the two alternatives, "on" sounds better for me too. @Girgias Do you have any preference?

Comment (Girgias) [reply reference/sqlite3/sqlite3result/finalize.xml:35]: No real preference here, but  quick `git grep` seems to show that "called on" is more common for this situation.

File: reference/sqlite3/sqlite3result/finalize.xml [hunk 2]
```diff
@@ -25,10 +25,40 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   Returns &true;.
+   &return.true.always;.
```
Comment (Girgias) [inline reference/sqlite3/sqlite3result/finalize.xml:28]:
```suggestion
   &return.true.always;
```

**Pull request merged**